CHANGELOG.md

```diff
@@ -13,6 +13,48 @@ next release
 
 ### UI Changes
 
+1.39.0 (2022-10-31)
+-------------------
+### Backend Changes
+
+#### ⛔ Breaking Changes
+
+#### New Features
+
+* Add support for OpenSearch 2.x ([@gaurav-05](https://github.com/gaurav-05) in [#3966](https://github.com/jaegertracing/jaeger/pull/3966))
+
+#### Bug fixes, Minor Improvements
+
+* Pin SBOM action to commit ([@yurishkuro](https://github.com/yurishkuro) in [bb49249](https://github.com/jaegertracing/jaeger/commit/bb492490594c9d9321ed9242862ac2a8864ff771))
+* Bump github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger ([@dependabot[bot]](https://github.com/apps/dependabot) in [#4009](https://github.com/jaegertracing/jaeger/pull/4009))
```

Comment:
Should we be listing all these dep bumps? I think they are just noise for release notes

Reply:
I have removed all dependabot PRs.